docs/tutorial_bracket_strategy.md

```diff
@@ -0,0 +1,224 @@
+
+You find the core script placing orders for the basic liquidity strategy in the folder `scripts/complete_liquidity_provision.js`.
+
+Before you start this script, you need to export some environment variables and prepare the parameters.
+
+### Env variables
+
+Run the following commands in your console:
+
+```ssh
+export NETWORK_NAME=rinkeby
+export GAS_PRICE_GWEI=<gas price>
+export MASTER_SAFE=<your safe address>
+export PK=<your private key of the proposer-account>
+```
+
+The `NETWORK_NAME` should be set to Mainnet or Rinkeby. The gas price should be set in a way that your transaction gets mined in a reasonable time. Check out reasonable gas prices on [ethgasstation](https://ethgasstation.info/).
+The `MASTER_SAFE` should be the safe address from the Gnosis-Safe created in the previous section.
+The `PK` should be the private key of your proposer account used as an owner in the Gnosis-Safe.
+Note, that in our current setup the PK is not secured with the highest standards.
+Hence, do not deposit more funds than some deployment ETH into this account.
+
+### Script parameters
+
+The script `complete_liquidity_provision` takes the following non-optional parameters:
+
+- _masterSafe_: This is the MASTER-SAFE you generated in the previous step. You should set the value to \$MASTER_SAFE
+- _fleetSize_: The fleet size determines how many brackets you want to deploy. This number must be even and smaller than 20.
+- _targetToken_: You are specifying two tokens you want to provide liquidity for, the _targetToken_ and the _stableToken_ via their indices. In order to get the `tokenIndex` of a token, follow the next section. If you specify the more stable token via stableToken and the more volatile token via targetToken, then most likely the price parameter is more intuitive. Eg., if you are trading `ETH/DAI` with `stableToken=Index(DAI)`, then the parameter `currentPrice` can be specified as a number > 1.
+- _stableToken_: specifies the second token you want to trade via its `tokenIndex`. In order to determine the tokenIndex, see the section.
+- _investmentTargetToken_: This is the cumulative sum of all targetTokens you want to invest in all brackets
+- investmentStableToken: This is the cumulative sum of all stableTokens you want to invest in all brackets
+- _currentPrice_: Please provide the current price of the trading pair: [ targetToken ] / [ stableToken ]. Your input will be checked for plausibility by the script via the price of dex.ag
+- _lowestLimit_: The liquidity provided will be split over the a price range of [lowestLimit, highestLimit]. Hence, the lowestLimit specifies the lowest price any bracket should trade.
+- _highestLimit_: The liquidity provided will be split over the a price range of [lowestLimit, highestLimit]. Hence, the highestLimit specifies the highest price any bracket should trade.
+
+#### Getting the `tokenIndex` of your tokens
```

Comment:
```suggestion
#### Getting the index of your tokens
```

Reply:
Also in the following section, I don't see a reason to call it "`tokenIndex`" instead of "token index" or just "index"